packages/core/js-sdk/src/admin/exchange.ts

```diff
@@ -73,72 +73,6 @@ export class Exchange {
     )
   }
 
-  async delete(
-    id: string,
-    query?: HttpTypes.SelectParams,
-    headers?: ClientHeaders
-  ) {
-    return await this.client.fetch<HttpTypes.AdminExchangeDeleteResponse>(
-      `/admin/exchanges/${id}`,
-      {
-        method: "DELETE",
-        headers,
-        query,
-      }
-    )
-  }
-
-  async addItems(
-    id: string,
-    body: HttpTypes.AdminAddExchangeItems,
-    query?: HttpTypes.SelectParams,
-    headers?: ClientHeaders
-  ) {
-    return await this.client.fetch<HttpTypes.AdminExchangeResponse>(
-      `/admin/exchanges/${id}/exchange-items`,
-      {
-        method: "POST",
-        headers,
-        body,
-        query,
-      }
-    )
-  }
-
-  async updateItem(
-    id: string,
-    actionId: string,
-    body: HttpTypes.AdminUpdateExchangeItem,
```

Comment:
nit: think that this types can be removed as well

Reply:
Good point! will remove them